WordPress/gutenberg
# RSS: Added Colour support

## What?

Add color  support to the RSS block.
Related to https://github.com/WordPress/gutenberg/issues/43245

## Why?

RSS block is missing color  support.

## How?
Adds the color support via block.json


## Testing Instructions

- Go to the Global Styles setting ( under appearance > editor > styles > edit styles > blocks ).
- Make sure that the RSS block's color is configurable via Global Styles.
- Verify that Global Styles are applied correctly in the editor and frontend.
- Edit the template/page, Add RSS block and apply the color for text, background and links .
- Verify that block color  styles take precedence over global block settings.
- Verify that block color display correctly in both the editor and frontend.

**NOTE**:
This PR is dependent on some style changes in another [PR](https://github.com/WordPress/gutenberg/pull/66411)

## Screenshots or screencast 

Backend:
![rsscolor](https://github.com/user-attachments/assets/c3cbdf21-f59f-41c3-97e8-6f60b7059d34)

Frontend:
![rsscolorfront](https://github.com/user-attachments/assets/78b50eaa-02b3-4bc3-ac2e-091297ffd27f)
